cfg/SpatioTemporalVoxelLayer.cfg

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
Did you add everything that's a "layer" parameter (vs. an observation source parameter) and default to the coded defaults?

Reply:
not yet I haven't review the reconfigure part yet.

Reply:
Problems encountered:
-  It's not possible to change in real-time any parameters belonging to observation sources though reconfigure... Like:

max_obstacle_height
obstacle_range
...

I think the only worth adding are:
transform_tolerance
mapping_mode 
map_save_duration

And with the last two I have some doubts, if they have any affect in real time or are only called in the init(). Can you clarify please?

Reply:
Yes that makes sense, I'm looking to just have everything [here](https://github.com/SteveMacenski/spatio_temporal_voxel_layer/blob/kinetic-devel/example/standard_indoor_environment_config.yaml#L18) and above - just the "layer" parameters, not the "observation" specific ones. We could have dynamic reconfigure actually reconfigure the buffer vectors as well but that's out of the scope of this PR and what you're trying to do

I think missing are
- mapping_mode
- map_save_duration